lib/tfenv-version-name.sh

```diff
@@ -10,7 +10,7 @@ function tfenv-version-name() {
       && log 'debug' "TFENV_VERSION_FILE retrieved from tfenv-version-file: ${TFENV_VERSION_FILE}" \
       || log 'error' 'Failed to retrieve TFENV_VERSION_FILE from tfenv-version-file';
 
-    TFENV_VERSION="$(cat "${TFENV_VERSION_FILE}" || true)" \
+    TFENV_VERSION="$(grep --invert-match '^#' "${TFENV_VERSION_FILE}" || true)" \
```

Comment:
Might be good to have some kind of  `tfenv-version-file-parse` function rather than repeat this everywhere?

Reply:
Seems perfectly reasonable. Would you rather incorporate that into this PR, or take it as it is for now?

Reply:
Done here now, in 71092b402d6ab4e6d6a2264aeed66c0e3e213902.